### PR TITLE
Move the file service config into a separate file

### PR DESCRIPTION
**Pull request by Windvis**

### CODE REVIEW

File: config/resources/file-service.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+  "version": "0.1",
+  "prefixes": {
+    "nfo": "http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#",
+    "dbpedia": "http://dbpedia.org/ontology/",
+    "nie": "http://www.semanticdesktop.org/ontologies/2007/01/19/nie#",
+    "dct": "http://purl.org/dc/terms/",
+    "skos": "http://www.w3.org/2004/02/skos/core#",
+    "org": "http://www.w3.org/ns/org#",
+    "vs": "http://www.w3.org/2003/06/sw-vocab-status/ns#",
```
Comment (anitacaron) [inline config/resources/file-service.json:10]: Clean up those prefixes?

Comment (Windvis) [reply config/resources/file-service.json:10]: Should I remove the `ext` one as well?

Comment (Windvis) [reply config/resources/file-service.json]: Done!

Comment (anitacaron) [reply config/resources/file-service.json]: You just deleted the vs, but the other two also don't need... :)

Comment (Windvis) [reply config/resources/file-service.json]: Time to call it a day :smile:. Sorry will fix.

**Review by anitacaron — APPROVED** [review]

**Pull request merged**